runtime/exec.cpp

```diff
@@ -69,7 +70,7 @@ static ExecStatus exec_impl(const string &cmd, const ExecHandler &handler) {
   return {true, ret, last_line};
 }
 
-static ExecStatus passthru_impl(const string &cmd) {
+ExecStatus passthru_impl(const string &cmd) {
```

Comment:
same about passthru_impl
```suggestion
ExecStatus passthru_impl(const string &cmd) {
  dl::CriticalSectionGuard heap_guard;

```